src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAroundBasic.java

```diff
@@ -1,277 +1,247 @@
 package com.google.checkstyle.test.chapter4formatting.rule462horizontalwhitespace;
 
 /**
- * Class for testing whitespace issues.
- * violation missing author tag
- **/
-class InputWhitespaceAroundBasic
-{
-    private int mVar1= 1; // violation ''=' is not preceded with whitespace.'
-    private int mVar2 =1; // violation ''=' is not followed by whitespace.'
-    /** Should be ok **/
-    private int mVar3 = 1;
-    int xyz;       // multiple space between content and double slash.
-    int abc; //       multiple space between double slash and comment's text.
-    int pqr;       //     testing both.
-
-    /** method **/
-    void method1()
-    {
-        final int a = 1;
-        int b= 1; // violation ''=' is not preceded with whitespace.'
-        b= 1; // violation ''=' is not preceded with whitespace.'
-        b +=1; // violation ''.*' is not followed by whitespace.'
-        b -=- 1 + (+ b); // violation ''-=' is not followed by whitespace.'
-        b = b++ + b--; // ok
-        b = ++ b - -- b; // ok
-    }
-
-    /** method **/
-    void method2()
-    {
-        synchronized(this) {
-        // 2 violations above:
-        //  ''synchronized' is not followed by whitespace.'
-        //  ''synchronized' is not followed by whitespace.'
-        }
-        try {// violation ''{' is not followed by whitespace.'
-        }
-        catch (RuntimeException e) {// violation ''{' is not followed by whitespace.'
-        }
-    }
-
-    /**
-       skip blank lines between comment and code,
-       should be ok
-    **/
-
-
-    private int mVar4 = 1;
-
-
-    /** test WS after void return */
-    private void fastExit()
-    {
-        boolean complicatedStuffNeeded = true;
-        if(!complicatedStuffNeeded)
-        // 2 violations above:
-        //  ''if' is not followed by whitespace.'
-        //  ''if' is not followed by whitespace.'
-        {
-            return; // should not complain about missing WS after return
-        }
-        else
-        {
-            // do complicated stuff
-        }
-    }
-
-
-    /** test WS after non void return
-     @return 2
-    */
-    private int nonVoid()
-    {
-        if (true)
-        {
-            return(2);
-            // 2 violations above:
-            //  ''return' is not followed by whitespace.'
-            //  ''return' is not followed by whitespace.'
-        }
-        else
-        {
-            return 2; // this is ok
-        }
-    }
-
-    /** test casts **/
-    private void testCasts()
-    {
-        Object o = (Object) new Object(); // ok
-        o = (Object) o; // ok
-        o = ( Object ) o; // ok
-        o = (Object)
```

Comment:
I think we should revert this as there are different cases with whitespace here.

Reply:
formatter changed method's location. we still have this method but formatter has removed the `(` & `)`:

https://github.com/checkstyle/checkstyle/blob/4f14fa2e584dda945cfbbb5e06e599522078e050/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAroundBasic.java#L87-L93

should I still add back the brackets?

Reply:
Yes, I think we should revert this block. Having `o = o;` following each other doesn't make sense

Reply:
done